web-registry/src/components/organisms/ProjectTopSection/ProjectTopSection.utils.ts

```diff
@@ -0,0 +1,37 @@
+const TOP_LINK_HEIGHT = 31;
+
+type GetTopLinksBoxHeightProps = {
+  creditClass: any;
+  creditClassVersion: any;
+  isVCSProject: boolean;
+  methodologyVersion: any;
+  registry: any;
```

Comment:
Had to use some any here because I couldn't get the types from the generated graphql file.
That being said we only care about checking these values' nullity here and not their actual values.